app/models/rubygem.rb

```diff
@@ -33,9 +33,11 @@ class Rubygem < ApplicationRecord
   after_create :create_gem_download
   before_destroy :mark_unresolved
 
-  MFA_RECOMMENDED_THRESHOLD = 165_000_000
+  MFA_RECOMMENDED_THRESHOLD = 150_000_000
```

Comment:
I don't think we should change this yet (maybe add a TODO to change it) as that would change the threshold upon deploy instead of the planned launch.